mrt/manager.py

```diff
@@ -330,13 +330,18 @@ def sync_cites_meetings(ctx):
 
 
 @cli.command(name='add_custom_field_sex')
+@click.option('--meeting', type=int)
 @click.pass_context
-def add_custom_field_sex(ctx):
+def add_custom_field_sex(ctx, meeting):
 # previously named add_custom_sex_field(ctx):
     app = ctx.obj['app']
 
     with app.test_request_context():
-        meetings = Meeting.query.all()
+
+        if meeting:
+            meeting_obj = Meeting.query.filter_by(id=meeting).scalar()
+
+        meetings = [meeting_obj] if meeting_obj else Meeting.query.all()
```

Comment:
If a meeting is not specified from the `--meeting` option, `meeting_obj` never gets declared and this will raise an error.